lib/DxilDia/DxcPixLiveVariables.cpp

```diff
@@ -148,6 +152,13 @@ void dxil_debug_info::LiveVariables::Impl::Init_DbgDeclare(
     return;
   }
 
+  VariableInfoMap *LiveVarInfoMap;
+  if (Variable->getName().startswith("global.")) {
```

Comment:
Why is the name start with "global."?
Any pass did that?

Reply:
Found it.

Reply:
ScalarReplAggregateHLSL's `PatchDebugInfo` did it.